File: cpp/gabaritos/gabarito13.cpp
#include <iostream>
#include <string>
#include <algorithm>
#include <cctype>

int main() {
    std::string str;
    std::cout << "Digite uma string: ";
    std::getline(std::cin, str);
    std::string normalizada;
    for (char c : str) {
        if (!std::isspace(c)) {
            normalizada += std::tolower(c);
        }
    }
    std::string reversa = normalizada;
    std::reverse(reversa.begin(), reversa.end());
    if (normalizada == reversa) {
        std::cout << "É um palíndromo." << std::endl;
    } else {
        std::cout << "Não é um palíndromo." << std::endl;
    }
    return 0;
}

File: csharp/gabaritos/gabarito13.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Digite uma string: ");
        string str = Console.ReadLine();
        string normalizada = str.ToLower().Replace(" ", "");
        string reversa = new string(normalizada.Reverse().ToArray());
        if (normalizada == reversa)
        {
            Console.WriteLine("É um palíndromo.");
        }
        else
        {
            Console.WriteLine("Não é um palíndromo.");
        }
    }
}